static/docs/user-guide/introduction.md

```diff
@@ -0,0 +1,31 @@
+Introduction
+------------
+
+
+The DVC user guide constitutes DVC files and directories, DVC file format, external
+dependencies, external outputs, update a tracked file, anonymized usage analytics which
+are part of basic. Customizations has DVC autocomplete, plugins, developement version
+and contributing guide.
+
+
+Let's understand the basic concepts as you use dvc.
+
+### Workspace
+
+Work space is the current work directory. The example workspace look as follows -
+
+### Cache
+
+DVC cache is a hidden (by default it's located in the `.dvc/cache` directory) storage.
+For files that are under DVC control it keeps them and their different versions. You
+can find more information on cache as described in this [doc](https://github.com/iterative/dvc.org/blob/master/static/docs/commands-reference/config.md#cache).
```

Comment:
link should go to dvc.org - something like `/doc/commands-reference/config` is enough.